src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -652,18 +620,17 @@ class FormatStringGenerator : public Generator<std::string> {
         : _rng{generatorArgs.rng}, _format{node["format"].maybe<std::string>().value_or("")} {
         std::stringstream msg;
         if (!node["format"]) {
-            msg << "Malformed FormatString: format missing '" << _format << "'" << node
-                << "\n";
+            msg << "Malformed FormatString: format missing '" << _format << "'" << node << "\n";
```

Comment:
I didn't not intentionally edit these. I have clang-format in my emacs and it runs on save.